### ISSUE TITLE
Support for password change in back office UI

### ISSUE THREAD

**Issue by jvelo** [mayocat/mayocat-shop#240]

**Comment by nesk** [comment]
One more thing to do with this issue is to improve responses for the `/api/user/` endpoint. Currently, if you create a user with a password which doesn't meet minimum requirements, the server responds with a 500 HTTP code and the following payload:

```json
{"message":"There was an error processing your request. It has been logged (ID d20c97197a0e31f7)."}
```

The only way to know the password is at fault is to read the server logs, not very _user-friendly_ ;)

**Comment by jvelo** [comment]
Right. It gives the correct error when creating a customer (https://github.com/mayocat/mayocat-shop/blob/master/shop/customer/web/src/main/groovy/org/mayocat/shop/customer/web/TenantCustomerAccountWebView.groovy#L179) but not when directly creating a user.
